bin/nginx_release_downloads.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
These scripts run in environments without `bash` (such as in Alpine), so I keep it POSIX.

Reply:
Ack, addressed in https://github.com/DataDog/nginx-datadog/pull/16/commits/57c3eedde296169327ad1d20b9b2d395b073a086